### PR TITLE
Add channel sink prefix

### PR DESCRIPTION
**Pull request by navidshaikh**

### CODE REVIEW

File: CHANGELOG.adoc [hunk 1]
```diff
@@ -12,6 +12,18 @@
 | https://github.com/knative/client/pull/[#]
 ////
 
+
+## v0.17.3 (2020-11-05)
```
Comment (navidshaikh) [inline CHANGELOG.adoc:16]: using today's date

**Review by navidshaikh — COMMENTED** [review]
/cc @dsimansk @rhuss

**Review by rhuss — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**